### PR TITLE
Update build statuses using access key

### PR DESCRIPTION
**Pull request by gorkat**
It is now possible to pass TuleapNotifyCommitStatusStep the id of an
access key to notify Tuleap.

### CODE REVIEW

File: languages/en/user-guide/ci.rst [hunk 1]
```diff
@@ -78,18 +78,26 @@ Pull request notification configuration
...
 
-Before using the command you will need a special, secret token to ensure
-that it's your jenkins job that recorded the build status. To do so, go into repository
-*Settings > CI Builds*:
+Before using the command, you will need to provide Jenkins a way to authenticate on Tuleap's side to ensure
+that it's your Jenkins's job that recorded the build status.
+
+There are currently two ways to proceed:
+ - Either `using the CI token`_ of the repository
+ - Either `using an access key`_ of a user belonging to a user group authorized to set build statuses.
```
Comment (Erwyn) [inline languages/en/user-guide/ci.rst:86]: [Either][Or][]

Comment (Erwyn) [inline languages/en/user-guide/ci.rst:86]: [Either][Or]

File: languages/en/user-guide/ci.rst [hunk 2]
```diff
@@ -102,18 +110,41 @@ In Jenkins, go to *Credentials > global > Add Credentials*, select the ``Secret
...
-                             credentialId: 'CRED_ID_OF_CI_TOKEN'
+                             credentialId: 'CRED_ID_OF_CI_TOKEN_OR_ACCESS_KEY'
 
 Where:
- - BUILD_STATUS is either 'failure' or 'success'.
- - GIT_REPOSITORY_ID is the ID of the concerned git repository. The ID can be found in the URL query (repo_id=XXX) in the 'Settings' menu of the repository.
- - CRED_ID_OF_CI_TOKEN is the Jenkins credential ID of the CI token. Here it is ``tuleap-ci-token``
+ - ``BUILD_STATUS`` is either 'pending', 'failure' or 'success'.
+ - ``GIT_REPOSITORY_ID`` is the ID of the concerned git repository. The ID can be found in the URL query (repo_id=XXX) in the 'Settings' menu of the repository.
+ - ``CRED_ID_OF_CI_TOKEN_OR_ACCESS_KEY`` is the Jenkins credential ID of the CI token OR the access key bound to your repository. In the example below, it is ``b9a4832d-f26d-46ca-acd7-15b935de9069``
```
Comment (Erwyn) [inline languages/en/user-guide/ci.rst:147]: I wouldn't say bound as it implies that somehow I would have linked the Access Key to my repository.

what about: « the access key of a user who has the permission to set builds statuses on your repository »

**Review by Erwyn — CHANGES_REQUESTED** [review]

File: languages/en/user-guide/ci.rst (hunk 1 above)
Comment (gorkat) [reply languages/en/user-guide/ci.rst:86]: agreed

**Review by Erwyn — APPROVED** [review]

**Pull request merged**